control-spec.txt

```diff
@@ -2233,6 +2233,8 @@
       MEASUREMENT_EXPIRED (As "TIMEOUT", except that we had left the circuit
                            open for measurement purposes to see how long it
                            would take to finish.)
+      IP_NOW_REDUNDANT    (Closing a circuit to an introduction point that
+                           has become redundant.)
```

Comment:
I would add one tiny detail which is that it is a circuit that was opened in parallel to others. In other words, it is redundant because it was built in parallel in the first place and another circuit won the "race".